CHANGELOG.md

```diff
@@ -1,5 +1,23 @@
 # Changelog
 
+## [2.1.0](https://github.com/voxpupuli/beaker-vmware/tree/2.1.0) (2024-04-23)
+
+[Full Changelog](https://github.com/voxpupuli/beaker-vmware/compare/2.0.0...2.1.0)
+
+**Implemented enhancements:**
+
+- Add Ruby 3.3 to CI matrix [\#43](https://github.com/voxpupuli/beaker-vmware/pull/43) ([bastelfreak](https://github.com/bastelfreak))
+- Add Ruby 3.2 support [\#33](https://github.com/voxpupuli/beaker-vmware/pull/33) ([bastelfreak](https://github.com/bastelfreak))
+
+**Merged pull requests:**
+
+- rubocop: Use task from voxpupuli-rubocop [\#42](https://github.com/voxpupuli/beaker-vmware/pull/42) ([bastelfreak](https://github.com/bastelfreak))
+- Replace dead dependency rbvmomi with rbvmomi2 [\#41](https://github.com/voxpupuli/beaker-vmware/pull/41) ([bastelfreak](https://github.com/bastelfreak))
```

Comment:
I think this deserves to be more prominent. Enhancement?

Reply:
updated it